doc/conf/YAML.md

```diff
@@ -0,0 +1,45 @@
+# Yaml - Configuration file
+
+To use read from Yaml file option it must has be installed.
```

Comment:
To use the Yaml conf file option it has to be installed.
To use the Yaml conf file option it must to be installed.